clickhouse_db_schema/misc.disabled_tests_historical/schema.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE misc.disabled_tests_historical
+(
+    `day` Date,
+    `timestamp` DateTime,
+    `name` String,
+    `issueNumber` Int32,
+    `platforms` Array(String),
+    `_meta` Tuple(
+        bucket String,
+        key String)
+)
+ENGINE = SharedReplacingMergeTree('/clickhouse/tables/{uuid}/{shard}', '{replica}')
+ORDER BY (day, name)
```

Comment:
Maybe also order by `issueNumber` as I can see the field is used in a JOIN in https://github.com/pytorch/test-infra/pull/6692